source/cellular_bg770.c

```diff
@@ -267,8 +267,8 @@ static bool tryBuildRATScanSequenceString( const BG770RATScanSequence_t * pRATSc
 
 static cellularModuleContext_t cellularBg770Context = { 0 };
 
-static bool configSkipPostHWFlowControlSetupIfChanged = false;
-static CellularModuleFullInitSkippedResult_t fullInitSkippedResult = CELLULAR_FULL_INIT_SKIPPED_RESULT_ERROR;
+static volatile bool configSkipPostHWFlowControlSetupIfChanged = false;
+static volatile CellularModuleFullInitSkippedResult_t fullInitSkippedResult = CELLULAR_FULL_INIT_SKIPPED_RESULT_ERROR;

```

Comment:
Are these modified by multiple tasks?

Reply:
They're not currently modified by multiple tasks but they could be so I'm just making it safe to be set/read by any thread.